zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -39,7 +40,7 @@ sealed trait Server[-R, +E] { self =>
 
 object Server {
   private[zhttp] final case class Config[-R, +E](
-    leakDetectionLevel: LeakDetectionLevel = LeakDetectionLevel.SIMPLE,
+    leakDetectionLevel: LeakDetectionLevel = LeakDetectionLevel.PARANOID,
```

Comment:
We shouldn't keep `PARANOID` as default. It affects performance quite badly and most people will deploy this on production without changing the configuration.